Makefile.toml

```diff
@@ -1185,3 +1185,13 @@ cat << EOF > src/config/example.toml
 EOF
 cargo run -p risingwave_common --bin example-config >> src/config/example.toml
 '''
+
+[tasks.psql]
+category = "RiseDev - PSQL Client"
+description = "Run local psql client with default connection parameters. You can pass extra arguments to psql."
+script = '''
+#!/usr/bin/env bash
+set -euo pipefail
+
+psql -h localhost -p 4566 -U root -d dev "$@"
```

Comment:
I guess it'll be better to avoid hardcoding host or port here. It seems that we already have the mechanism to read the configuration of the cluster launched by `risedev d`, which is originally designed for `risedev ctl`. Do you think we can reuse this for reading the address of the frontend node?

https://github.com/risingwavelabs/risingwave/blob/f3991cde5f9ff5e82b958ee71af1d38a4755d1ff/Makefile.toml#L501-L508
https://github.com/risingwavelabs/risingwave/blob/75efaa19ac5cd49aaa0c0354ad3c9509f7a67eec/src/risedevtool/src/risectl_env.rs#L22-L23